packages/create-nx-workspace/src/utils/nx/ab-testing.ts

```diff
@@ -1,26 +1,54 @@
 import { isCI } from '../ci/is-ci';
 
+export const NxCloudChoices = ['yes', 'github', 'circleci', 'skip'];
+
 const messageOptions = {
-  nxCloudCreation: [
+  setupCI: [
+    {
+      code: 'enable-nx-cloud',
+      message: `Do you want Nx Cloud to make your CI fast?`,
+      initial: 'github',
+      choices: [
+        { value: 'yes', name: 'Yes, enable Nx Cloud' },
+        { value: 'github', name: 'Yes, configure Nx Cloud for GitHub Actions' },
+        { value: 'circleci', name: 'Yes, configure Nx Cloud for Circle CI' },
+        { value: 'skip', name: 'Skip for now' },
+      ],
+      fallback: undefined,
+    },
     {
-      code: 'set-up-distributed-caching-ci',
-      message: `Enable remote caching to make your CI faster`,
+      code: 'set-up-ci',
+      message: `Set up CI with caching, distribution and test deflaking`,
+      initial: 'github',
+      choices: [
+        { value: 'github', name: 'Yes, for GitHub Actions with Nx Cloud' },
+        { value: 'circleci', name: 'Yes, for CircleCI with Nx Cloud' },
+        { value: 'skip', name: 'Skip for now' },
+      ],
+      fallback: { value: 'skip', key: 'setupNxCloud' },
     },
   ],
-  nxCloudMigration: [
+  setupNxCloud: [
     {
-      code: 'make-ci-faster',
-      message: `Enable remote caching to make your CI faster?`,
+      code: 'enable-caching',
+      message: `Would you like remote caching to make your build faster`,
```

Comment:
The other question had a question mark

```suggestion
      message: `Would you like remote caching to make your build faster?`,
```

Open question, is “build” too specific? Should we say “tasks” instead? Or “builds, tests etc”

Reply:
We had these discussions in the past. I was also on `task` team, as it's more accurate but the reason for going with build is that it's historically the pain point people care about.